modules/@apostrophecms/schema/ui/apos/logic/AposSchema.js

```diff
@@ -120,7 +125,7 @@ export default {
               required
             },
             value: {
-              data: this.modelValue[item.name]
+              data: this.modelValue[item.name] || {}
```

Comment:
Are we  sure data is always supposed to be an object here?
Should be `this.modelValue.data[item.name]`. Looks like it's not used we should verify and create a ticket if needed.